django/template/context.py

```diff
@@ -210,28 +221,26 @@ def __init__(self, request, dict_=None, processors=None,
         self._processors_index = len(self.dicts)
         self.update({})         # placeholder for context processors output
 
-    @property
-    def template(self):
-        return self._template
-
-    @template.setter
-    def template(self, template):
-        # Execute context processors when Template.render(self, context) sets
-        # context.template = self. Until then, since the context isn't tied to
-        # an engine, it has no way to know which context processors to apply.
-        self._template = template
-        if hasattr(self, '_processors_index'):
-            if template is None:
-                # Unset context processors.
-                self.dicts[self._processors_index] = {}
-            else:
-                # Set context processors for this engine.
-                processors = (template.engine.template_context_processors +
-                              self._processors)
-                updates = {}
-                for processor in processors:
-                    updates.update(processor(self.request))
-                self.dicts[self._processors_index] = updates
+    @contextmanager
+    def bind_template(self, template):
```

Comment:
Er, never mind. Of course you do - failed to see the large chunk of collapsed context in between.